recipe/meta.yaml

```
@@ -1,32 +1,31 @@
 {% set name = "rich" %}
-{% set version = "12.5.1" %}
-
+{% set version = "13.3.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 63a5c5ce3673d3d5fbbf23cd87e11ab84b6b451436f1b7f19ec54b6bc36ed7ca
+  sha256: 2d11b9b8dd03868f09b4fffadc84a6a8cda574e40dc90821bd845720ebb8e89c
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  skip: True  # [py<36]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation
+  skip: True  # [py<37]
 
 requirements:
   host:
     - pip
     - setuptools
     - wheel
     - python
-    - poetry >=1.0.10
-    - poetry-core
+    - poetry-core >=1.0.10
   run:
+    - markdown-it-py >=2.2.0,<3.0.0
     - commonmark >=0.9.0,<0.10.0
     - dataclasses >=0.7,<0.9
-    - pygments >=2.6.0,<3.0.0
+    - pygments >=2.13.0,<3.0.0
     - python
     - typing_extensions >=4.0.0,<5.0.0
```

```suggestion
    - typing_extensions >=4.0.0,<5.0.0. # [py<39]
```
https://github.com/Textualize/rich/blob/v13.3.5/pyproject.toml#LL30C28-L30C28

---

Thanks!